modules/mediawiki/files/bin/mwscript.py

```diff
@@ -6,7 +6,7 @@
 if len(sys.argv) < 3:
     raise Exception("Not Enough Parameters")
 script = sys.argv[1]
-if script == 'importDump.php':
+if script in ['importDump.php', 'deleteBatch.php', 'importImages.php', 'rebuildall.php']:
```

Comment:
Isn't a tuple slightly more efficient in this case? Correct me if I'm wrong though. Also can this be a variable to make changing it later on a bit easier?

Reply:
honestly, no idea on tuple v list.

As to changing it later. I'll change a lot when I add python CI in like a week so not worried.

Reply:
Can this be merged then if your not planning to change that yet?

Reply:
Yes